code/datums/mind/antag.dm

```diff
@@ -202,7 +202,7 @@
 	current.faction |= creator.faction
 	creator.faction |= current.faction
 
-	if(creator.mind.special_role)
+	if(creator.mind?.special_role)
```

Comment:
similar

Reply:
in the case the creator of the holoparasite's creator not having a mind, to prevent runtimes